infra/cleanup.sh

```diff
@@ -34,3 +34,7 @@ for compute_name in "${amlcompute_to_delete[@]}"; do
   fi
 done
 
+# delete registry of yesterday
+RegistryToBeDeleted=DemoRegistry$(date -d '-1 days' +'%m%d')
+echo "Deleting registry $RegistryToBeDeleted"
+az resource delete -n $RegistryToBeDeleted --resource-type Microsoft.MachineLearningServices/registries
```

Comment:
Have you tested this locally? Cause I got an example of deleting a registry using `az resource delete` with `-g <ResourceGroup>`. Don't know if it can work without a resource group name.

Reply:
Seems it's a no-op without -g argument.  Updated.  Thanks!